core/packages/test/scripts/set-env.sh

```diff
@@ -11,9 +11,9 @@ lodestar_version="${LODESTAR_VER:-1.2.2}"
 contract_dir="$core_dir/packages/contracts"
 relay_dir="$root_dir/relayer"
 relay_bin="$relay_dir/build/snowbridge-relay"
-output_dir=/tmp/snowbridge
-output_bin_dir="$output_dir/bin"
-ethereum_data_dir="$output_dir/geth"
+export output_dir=/tmp/snowbridge
+export output_bin_dir="$output_dir/bin"
+export ethereum_data_dir="$output_dir/geth"
```

Comment:
Why do we need to export `ethereum_data_dir`? I see we're using `output_dir` and `output_bin_dir` in the zombienet config, but can't find a use of `ethereum_data_dir`

Reply:
It is used to launch a local eth node. 
https://github.com/Snowfork/snowbridge/pull/754/files#diff-d6b5dd8f20f1dcdd2b05128403e3822304831554c07c815c8e94206a9f1cb567R11-R12

Reply:
yes, though we use it we do need to export this param. But seems not a big issue will not hurt? Anyway just change it.